src/main/java/seedu/address/logic/relationship/DeleteRelationshipCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.relationship;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import seedu.address.model.person.Person;
+import seedu.address.model.person.relationship.Relationship;
+import seedu.address.model.person.relationship.RelationshipManager;
+
+public class DeleteRelationshipCommand {
+    private final Map<String, Person> personMap;
+    private final RelationshipManager relationshipManager;
+
+    /**
+     * Creates a new DeleteRelationshipCommand with the given person map and relationship manager.
+     *
+     * @param personMap           A map linking UUID strings to Person objects.
+     * @param relationshipManager The RelationshipManager instance managing the relationships.
+     */
+    public DeleteRelationshipCommand(Map<String, Person> personMap, RelationshipManager relationshipManager) {
+        this.personMap = personMap;
+        this.relationshipManager = relationshipManager;
+    }
+
+    /**
+     * Parses and executes the given delete relationship command.
+     *
+     * @param command The delete relationship command to parse and execute.
+     * @throws IllegalArgumentException If the command is invalid.
+     */
+    public void parseCommand(String command) {
+        String[] commandParts = command.split("\\s+");
+
+        if (commandParts.length != 4) {
+            throw new IllegalArgumentException("Invalid command format");
+        }
+
+        String relationType = commandParts[1].substring(1);
+        String[] personUuids = commandParts[2].split(",");
+        if (personUuids.length != 2) {
+            throw new IllegalArgumentException("Invalid command format");
+        }
+
+        String uuid1 = matchUuid(personUuids[0]);
+        String uuid2 = matchUuid(personUuids[1]);
+
+        List<Relationship> relationships = relationshipManager.getRelationships(relationType);
+        if (relationships.isEmpty()) {
```

Comment:
This check is redundant cos if the relationship list is empty, it will just skip the loop after this and return the same IllegalArgumentException anyways